src/app/qgsapplayertreeviewmenuprovider.cpp

```diff
@@ -348,7 +358,7 @@ QMenu *QgsAppLayerTreeViewMenuProvider::createContextMenu()
         if ( !layer->isInScaleRange( mCanvas->scale() ) )
           menu->addAction( tr( "Zoom to &Visible Scale" ), QgisApp::instance(), &QgisApp::zoomToLayerScale );
 
-        QMenu *menuSetCRS = new QMenu( tr( "&Layer CRS" ), menu );
+        QMenu *menuSetCRS = new QMenu( tr( "La&yer CRS" ), menu );
```

Comment:
This is nitpicking, but I was thinking that Y is the last candidate for *Styles* that is currently missing a shortcut and it might be useful to provide a shortcut way to copy and paste styles. It is the last candidate for *Layer CRS* too though... What do you think?

Reply:
I'd say we drop the shortcut for CRS entirely?

Reply:
I'm ok with it!

Reply:
Done